src/components/vm/filesystems/vmFilesystemsCard.jsx

```diff
@@ -238,10 +238,10 @@ const VmFilesystemAddModal = ({ connectionName, setIsOpen, objPath, vmName, vmSt
                                    isExpanded={additionalOptionsExpanded}>
                     <FormGroup hasNoPaddingTop
                                fieldId={`${idPrefix}-modal-xattr`}
-                               label="Extended attributes">
+                               label="Options">
```

Comment:
I think original `Extended attributes` is just fine. There are more options which may be added in the future for shared directories such as `cache mode`, `lock`, so lavelling only Extended attributes as "options" can be confusing.

Reply:
Ok, I also just noticed the label was not translated so fixed that as well.